tfe/data_source_outputs.go

```diff
@@ -159,19 +159,28 @@ type outputData struct {
 
 func (d dataSourceOutputs) readStateOutput(ctx context.Context, tfeClient *tfe.Client, orgName, wsName string) (*stateData, error) {
 	log.Printf("[DEBUG] Reading the Workspace %s in Organization %s", wsName, orgName)
-	opts := &tfe.WorkspaceReadOptions{
-		Include: []tfe.WSIncludeOpt{tfe.WSOutputs},
-	}
-	ws, err := tfeClient.Workspaces.ReadWithOptions(ctx, orgName, wsName, opts)
+	ws, err := tfeClient.Workspaces.Read(ctx, orgName, wsName)
 	if err != nil {
 		return nil, fmt.Errorf("Error reading workspace: %w", err)
 	}
 
+	svOpts := &tfe.StateVersionCurrentOptions{
+		Include: []tfe.StateVersionIncludeOpt{tfe.SVoutputs},
+	}
+	sv, err := tfeClient.StateVersions.ReadCurrentWithOptions(ctx, ws.ID, svOpts)
```

Comment:
If you are listing the outputs in the next step, you probably don't need to include the outputs in the sv result.